src/config/dynamic-theme-fixes.config

```
@@ -7638,8 +7638,11 @@ i.transfer-progress-icon
 span.top-icon.language
 
 CSS
-.files-grid-view.fm .column-settings {
-    background-color: ${#181a1b} !important;
+.grid-wrapper {
+    outline: 0px !important;
```

Maybe changing the border color is bettter than removing it completely!
```suggestion
    outline-color: var(--darkreader-bg--surface-main);
```

---

I agree with this one.

---

Yup, even better!

---

Great!, Thanks.